.github/workflows/ci.yml

```
@@ -30,131 +30,4 @@ jobs:
-        .tox/sample_data/
-      cache-key: docs-${{ github.run_id }}
-      libraries: |
-        apt:
-          - libopenjp2-7
-          - graphviz
-      envs: |
-        - linux: build_docs
-
-  online:
-    if: "!startsWith(github.event.ref, 'refs/tags/v')"
-    needs: [docs]
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@main
-    with:
-      default_python: '3.8'
-      submodules: false
-      coverage: codecov
-      toxdeps: tox-pypi-filter
-      posargs: -n auto --dist loadgroup
-      libraries: |
-        apt:
-          - libopenjp2-7
-      envs: |
-        - linux: build_docs-gallery
-          posargs: ''
-          pytest: false
-          cache-path: |
-            docs/_build/
-            docs/generated/
-            .tox/sample_data/
-          cache-key: docs-${{ github.run_id }}
-          libraries:
-            apt:
-              - libopenjp2-7
-              - graphviz
-        - linux: py39-online
-
-  cron:
-    if: |
-      github.event_name == 'workflow_dispatch' || (
-        github.event_name == 'pull_request' &&
-        contains(github.event.pull_request.labels.*.name, 'Run cron CI')
-      )
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@main
-    with:
-      default_python: '3.8'
-      submodules: false
-      coverage: codecov
-      toxdeps: tox-pypi-filter
-      posargs: -n auto
-      libraries: |
-        apt:
-          - libopenjp2-7
-      envs: |
-        - linux: base_deps
-        - linux: py39-devdeps
-        - linux: py39-conda
-          libraries: ''
-
-  publish:
-    # Build wheels when pushing to any branch except main
-    # publish.yml will only publish if tagged ^v.*
-    if: |
-      (
-        github.event_name != 'pull_request' && (
-          github.ref_name != 'main' ||
-          github.event_name == 'workflow_dispatch'
-        )
-      ) || (
-        github.event_name == 'pull_request' &&
-        contains(github.event.pull_request.labels.*.name, 'Run publish')
-      )
-    needs: [test]
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@main
-    with:
-      test_extras: 'dev'
-      test_command: 'pytest -p no:warnings --doctest-rst -m "not mpl_image_compare" --pyargs sunpy'
-      submodules: false
-      targets: |
-        - cp3{8,9,10}-manylinux*_x86_64
-        - cp3{8,9,10}-macosx_x86_64
-        - cp3{8,9,10}-macosx_arm64
-    secrets:
-      pypi_token: ${{ secrets.pypi_token }}
-
-  notify:
-    if: always() && github.event_name == 'workflow_dispatch'
-    needs: [publish, online, cron]
-    runs-on: ubuntu-latest
-    steps:
-      - uses: Cadair/matrix-notify-action@main
-        with:
-          matrix_token: ${{ secrets.matrix_access_token }}
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          homeserver: ${{ secrets.matrix_homeserver }}
-          roomid: '!JYqfIVJjWANcHnfktY:cadair.com'
-          ignore_pattern: '.*Load.*'
+        - linux: 3.11-dev
```

```suggestion
        - linux: py311
          python-version: '3.11-dev'
```
That should work (https://github.com/OpenAstronomy/github-actions-workflows/issues/53)